doc/source/parameters/dem/insertion_info.rst

```diff
@@ -0,0 +1,41 @@
+Insertion Info
+-------------------
+Particle insertion information is defined in this section. This information includes ``insertion method``, ``inserted number of particles at each insertion step``, ``insertion frequency``, dimensions of the insertion box, approximate initial distance between particles at insertion steps, and information about the randomness of initial positions of particles. If the insertion method is ``uniform``, the particles are uniformly (without randomness in initial location) inserted. On the other hand, in ``non_uniform`` insertion, initial locations of particles in `x` and `y` directions are chosen randomly. Using ``non_uniform`` insertion, insertion ``random number range`` and ``insertion random number seed`` are also required in the parameter handler file.
+
+.. note::
+    Insertion in Lethe-DEM starts inserting particles from type 0 and proceeds to the next type when all the particles from the previous type are inserted.
+
+.. note::
+    If the insertion box is not large enough to insert ``inserted number of particles at each time step``, Lethe-DEM prints a warning to inform the user about the number of particles that can fit in the insertion box.
+
+.. note::
+    ``insertion distance threshold`` determines the initial distance between the particles in the insertion phase. As a result, it must be larger than 1 to avoid any initial collision between the inserted particles. It should also be compatible with the ``random number range``; especially if the ``random number range`` is large, a large value should be defined for ``insertion distance threshold``. Generally, we recommend users to use a value in the range of 1.3-2 (depending on the value of ``random number range``) for ``insertion distance threshold``.
```

Comment:
Can you give the mathematical definition of the insertion distance? Like it uses the threshold multiplied by the max particle diameter no?
I would add this explanation below after when you explain each parameter.

Reply:
It is added to the end of insertion_info.srt.

Reply:
I think its also worth mentioning that that insertion box should be smaller than the simulation domain for spherical and cylindrical domains and smaller or equal to the simulation domain for square or rectangular domains. I had a lot of problems defining it when I started using DEM.